client/src/pages/profile/Profile.jsx

```diff
@@ -115,9 +129,12 @@ export default function ProfilePage({userData, isLoggedIn}) {
                         <Typography variant="h3" gutterBottom component="div">
                         Reviews
                         </Typography>
-                        <Stack sx={{display: 'flex', alignItems: 'center', p: 1, m: 1,}}>
+                        <Stack sx={{ alignItems: 'center', p: 1, m: 1,}}>
                             <Button variant="contained" onClick = {handleReview}>Create Review</Button>
                         </Stack>
+                        <Stack sx={{ alignItems: 'center', p: 1, m: 1,}}>
+                       <Select options={reviewFilterOptions} defaultValue={reviewFilterOptions[0]}   onChange={handleFilter}/>
```

Comment:
Is there an MUI Select that can be used here instead for consistency? What are the perks of the the React select component?

Reply:
Yea I can change to MUI Select for consistency.
Reason i did react select is its clean one liner, whereas MUI is like 10 lines for same thing.